databricks/koalas/indexes.py

```diff
@@ -2498,6 +2518,78 @@ def __getattr__(self, item: str) -> Any:
                 return partial(property_or_func, self)
         raise AttributeError("'MultiIndex' object has no attribute '{}'".format(item))
 
+    def _get_level_number(self, level):
+        """
+        Return the level number if a valid level is given.
+        **this is an internal method**
```

Comment:
No need to document this because leading `_` already means that.

Reply:
Agree, I'm gonna remove this